packages/core/src/api.js

```diff
@@ -78,6 +78,13 @@ export function createPercyServer(percy, port) {
     .route('get', '/percy/dom.js', (req, res) => {
       return res.file(200, PERCY_DOM);
     })
+    // returns widths used in config and of devices enabled
+    .route('get', '/percy/widths', (req, res) => res.json(200, {
```

Comment:
This is an extra call - all SDKs already make healthcheck call, if we just return it as part of that we can just save it in class variables as this will be only required once and will remain same for all snapshots.